### PR TITLE
Update instruction for deploy perf-dash

### PR DESCRIPTION
**Pull request by mm4tt**

### CODE REVIEW

File: perfdash/README.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ Application server runs as a deployment on kubernetes cluster. It is hosted on
   one specified in the `Makefile`
 * Submit a PR, get required approvals and wait until it's merged
 * Run `make push` to push new image to container registry
-* Run `deploy.sh`
+* Refer to [perf-dash.k8s.io/README.md](https://github.com/kubernetes/k8s.io/blob/master/perf-dash.k8s.io/README.md) for the deploy instructions
+  
+  Note: Since 2020-04-21, perf-dash.k8s.io is hosted in the public **aaa** cluster.
```
Comment (oxddr) [inline perfdash/README.md:34]: Is this an actual name of the cluster or just a placeholder?

Comment (oxddr) [reply perfdash/README.md:34]: Actually, it is. Can you please add `(sic!)` - there is a huge surprise factor here.

Comment (mm4tt) [reply perfdash/README.md]: Done

**Comment by oxddr** [comment]
/lgtm

Thanks!

**Pull request merged**